Library/Formula/openjpeg.rb

```diff
@@ -22,7 +22,13 @@ class Openjpeg < Formula
   depends_on "libpng"
 
   def install
-    system "cmake", ".", *std_cmake_args
+    args = std_cmake_args
+    system "cmake", ".", *args
+    system "make", "install"
```

Comment:
It's more that not everyone will want/need the static library, and making them go through the build process twice for it feels a bit heavy-handed. I don't know. I'll let the other maintainers chime in.